lib/saml20.js

```diff
@@ -79,8 +79,9 @@ exports.create = function(options, callback) {
   sig.signingKey = options.key;
   
   sig.keyInfoProvider = {
-    getKeyInfo: function () {
-      return "<X509Data><X509Certificate>" + cert + "</X509Certificate></X509Data>";
+    getKeyInfo: function (key, prefix) {
+      prefix = prefix ? prefix + ':' : prefix;
```

Comment:
`prefix` is a string, added a typeof validation (and test) to make sure it's a string.